error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/IsInstanceLambdaUsage.java

```diff
@@ -39,7 +39,9 @@ public IsInstanceLambdaUsage() {}
 
   @Override
   public Description matchLambdaExpression(LambdaExpressionTree tree, VisitorState state) {
-    if (tree.getKind() != Kind.LAMBDA_EXPRESSION || tree.getBody().getKind() != Kind.INSTANCE_OF) {
+    if (tree.getKind() != Kind.LAMBDA_EXPRESSION
+        || tree.getBody().getKind() != Kind.INSTANCE_OF
```

Comment:
Side-remark: at some point we should take a stance around testing the `Kind` versus performing an `instanceof` check. I expect the former to be faster (didn't check), but the latter is easier to understand for new users. Also, once we set JDK 17 as a baseline we may naturally start to use the latter i.c.w. pattern matching.